omod/src/main/webapp/pages/metadata/locations/location.gsp

```diff
@@ -9,7 +9,7 @@
 
     def parentLocationOptions = []
     existingLocations.each {
-        parentLocationOptions.push([ label: ui.escapeJs(ui.format(it)), value: it.id ])
+        parentLocationOptions.push([ label: ui.encodeJavaScript(ui.format(it)), value: it.id ])
```

Comment:
thanks @jnsereko  have you tested this on a running openmrs instance. Add screen shots of before and after your changes

Reply:
Hello @HerbertYiga 
I have tested hid feature with OpenMRS sdk and the ui doesn't change but makes OpenMRS more secure.
![image](https://user-images.githubusercontent.com/58003327/122103470-5167d780-ce1f-11eb-8bb9-14d5dbc93a96.png)

Reply:
could you try to see on the different pull requests that  have been merged for solving XSS vulnerabilities and see how testing has been done

Reply:
@jnsereko another thing (besides making sure that the UI isn't broken) is to make sure that the original XSS vulnerability is still patched. So, for example, create a location with name `<script>alert(1)</script>` and make sure the alert doesn't show on the list locations page.

Otherwise, I think this looks good.

Reply:
> `<script>alert(1)</script>` and make sure the alert doesn't show on the list locations page.

thanks @HerbertYiga  and @isears for looking at this. 
I have created a location with the name `<script>alert(1)</script>`  but i am not getting the alert displaying `1`
So the script is not executed.

Reply:
> `<script>alert(1)</script>` and make sure the alert doesn't show on the list locations page.

thanks @HerbertYiga  and @isears for looking at this. 
I have created a location with the name `<script>alert(1)</script>`  but i am not getting the alert displaying `1`
So the script is not executed. 

now i don't know why Maven build is rejecting my code.

Reply:
I don't think the build errors are related to this PR.